styles/main.css

```diff
@@ -120,6 +141,21 @@ a {
   box-shadow: 0 0 5px rgba(58, 57, 57, 0.137);
 }
 
+#recent-work-modal .links a {
+  background-color: #36b37e;
+  color: #fff;
+  outline: none;
+  border: none;
+  padding: 10px 0;
+  font-size: 1.1rem;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 3px;
+  width: 45%;
+  margin: 0 2%;
+}
```

Comment:
- The buttons still break between the widths of 992px and 1241px. Kindly address this issue for a better user interface.

<img width="628" alt="image" src="https://user-images.githubusercontent.com/77190203/156329105-a52c6472-80e1-484e-b1fd-063de0cdb594.png">

Reply:
@medaminedev66 @Kingstalux Thanks, guys. I'm grateful for the review. I have a question though. As you can see in the screenshot it is working fine on my computer. Is there any chance this could happen because of different screen types or some other factor?
![993](https://user-images.githubusercontent.com/69077061/156348206-da8ec360-73de-4138-a375-d95226089270.png)
![1241](https://user-images.githubusercontent.com/69077061/156348216-16b00712-5743-4587-a1e8-fa75d9fb678c.png)

Reply:
Hello @Tadesse-Alemayehu, 
Sometimes your project on Github might be different from that on your machine so maybe try cloning your repo afresh from Github and you might be able to correct it then.

Reply:
Okay @Kingstalux. 
I'll try that. Thank you.

Reply:
@Kingstalux I have added a close button for the mobile display https://github.com/Tadesse-Alemayehu/Portfolio/pull/14/commits/2e9d3d9224fcf6da0c0b2a2bc61ba5b8bc9c04f1